File: code-examples/csharp/products/delete.cs
var client = new RestClient("http://api.marketcloud.it/v0/products3");
var request = new RestRequest(Method.DELETE);
request.AddHeader("authorization", "YourPublicKey:YourToken");
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
IRestResponse response = client.Execute(request);

File: code-examples/java/products/delete.java
HttpResponse<String> response = Unirest.delete("http://api.marketcloud.it/v0/products3")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .header("authorization", "YourPublicKey:YourToken")
  .asString();